Exclude `vitest.config.ts` from TypeScript Compilation

## Problem
The TypeScript build fails due to the file `vitest.config.ts` not being within the `rootDir` specified in the `tsconfig.json`. The error is as follows:

```
error TS6059: File '/home/runner/work/npm-audit-action/npm-audit-action/vitest.config.ts' is not under 'rootDir' '/home/runner/work/npm-audit-action/npm-audit-action/src'. 'rootDir' is expected to contain all source files.
```

## Solution
Exclude the `vitest.config.ts` file from the TypeScript compilation by modifying the `tsconfig.json` file.

## Suggested Changes
Update the `tsconfig.json` file as follows:
```
{
  "include": ["src/**/*"],
  "exclude": ["node_modules", "dist", "vitest.config.ts"]
}
```
This change ensures that `vitest.config.ts` is ignored by the TypeScript compiler while keeping the rest of the project structure intact.

## Benefits
- Prevents build failures caused by files outside the `rootDir`.
- Keeps the TypeScript configuration clean and simple.
- Maintains the integrity of the project structure.

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -8,5 +8,5 @@
     "noImplicitAny": true,                    /* Raise error on expressions and declarations with an implied 'any' type. */
     "esModuleInterop": true                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
   },
-  "exclude": ["node_modules", "**/*.test.ts"]
+  "exclude": ["node_modules", "**/*.test.ts", "vitest.config.ts"]
 }

